### ISSUE TITLE
SIGABRT crash in download-manager

### ISSUE THREAD

**Issue by hdasch** [atlas-engineer/next#696]
**Describe the bug**
Build on master @7188020c crashes when a ".pdf" URL is requested.
**Precise recipe to reproduce the issue**
`next --no-init --no-session https://www.gnu.org/software/emacs/manual/pdf/eintr.pdf`
**Information**
- OS name+version:
Linux (Artix - running release)
- Kernel name+version:
5.5.10-artix1-1
- Graphics card and driver:
Card: Intel Corporation HD Graphics 530 (rev 06)
Driver: xorg intel_drv.so version 2.99.917
- Desktop environment / Window manager name+version:
i3-gaps 4.18-1
- Next version (from =M-x next-version= or =next --version=):
1.5.0-7188020c
- How you installed Next (Guix pack, package manager, build from source):
Built from source
- Lisp implementation/version (if built from source):
SBCL 2.0.3
- WebKitGTK+ | QtWebEngine version:
webkit2gtk 2.28.0-3
About the WebKitGTK+ version:  it's _not_ 4.0, current version as of March 2020
is 2.28.

**Output when started from a shell**
```
<INFO> [09:39:16] "Loading global history of 739 URLs."
<INFO> [09:39:16] "Loading: https://www.gnu.org/software/emacs/manual/pdf/eintr.pdf."
<INFO> [09:39:17] Buffer #<GTK-BUFFER {1008070413}> downloads https://www.gnu.org/software/emacs/manual/pdf/eintr.pdf
<INFO> [09:39:17] DOWNLOAD-MANAGER::URI: "https://www.gnu.org/software/emacs/manual/pdf/eintr.pdf" DOWNLOAD-MANAGER::ARGS: (:DIRECTORY
                                                                                                                            #P"/home/hugh/Downloads/"
                                                                                                                            :COOKIES
                                                                                                                            ""
                                                                                                                            :PROXY
                                                                                                                            NIL)
<INFO> [09:39:18] Downloading
  https://www.gnu.org/software/emacs/manual/pdf/eintr.pdf
to
  '/home/hugh/Downloads/eintr.pdf'.

<INFO> [09:39:18] "Certificate host whitelist set to NIL for #<GTK-BUFFER {1007AEC843}>."
fatal error encountered in SBCL pid 805(tid 0x7fafb5bbf700):
SIGABRT received.

   0: Foreign function gsignal, pc = 0x7fc82d2e0ce5, fp = 0x7fc8202e8180
```

**Comment by Ambrevar** [comment]
Thanks for the report, I'll look into it as soon as possible.
I can't find commit 7188020c, which repo / commit are you using?

**Comment by hdasch** [comment]
I'm running github master.  HEAD:
```
commit 7188020cc16b48ba94ac054769ecac2649b4cabe (HEAD -> github, github/master)
Author: John Mercouris <john@mercouris.email>
Date:   Sat Apr 25 17:33:11 2020 +0200

    element-hint.lisp: reindentation

```

**Comment by Ambrevar** [comment]
Oops, I had forgotten to pull!  Thanks for the heads up!

**Comment by Ambrevar** [comment]
Fixed in 4355573521f3f8a80514479f7051e60a62da132c.
Thanks a lot for reporting!
